aderyn/src/main.rs

```diff
@@ -204,14 +217,84 @@ fn main() {
                     skip_update_check: args.skip_update_check,
                     stdout: args.stdout,
                 };
-                driver::drive_with(new_args, subscriptions);
+                if cmd_args.watch {
+                    println!("INFO: Aderyn is entering watch mode !");
+                    // setup debouncer
+                    let (tx, rx) = std::sync::mpsc::channel();
+
+                    // no specific tickrate, max debounce time 2 seconds
+                    let mut debouncer = new_debouncer(Duration::from_secs(2), None, tx).unwrap();
```

Comment:
Changing this value doesn't seem to change the refresh rate.
Values used: 0, 2, 5. The refresh duration _seems_ the same.

Does debounce time only kick in when many files are being changed, or just one?
It could be that the delay is just the compilation framework and aderyn running.

Reply:
In my understanding, debounce kicks in when it observes a period of 2 seconds when there are no file changes irrespective of how many they are. (1 or bulk)

Reply:
But I need to look more into since you said the refresh rate doesn't seem to change

Reply:
Plan for now is to schedule an interval of 2 seconds then read from report.json when we see event  `vscode/textDocumentSaved` (I don't remember the name exactly)

Reply:
There are 2 places where we declare 2 seconds. Did you try changing in both places?